mem0/embeddings/vllm.py

```diff
@@ -0,0 +1,45 @@
+import os
+from typing import Literal, Optional
+
+from openai import OpenAI
+
+from mem0.configs.embeddings.base import BaseEmbedderConfig
+from mem0.embeddings.base import EmbeddingBase
+
+
+class VllmEmbedding(EmbeddingBase):
+    def __init__(self, config: Optional[BaseEmbedderConfig] = None):
+        super().__init__(config)
+
+        # Default embedding model
+        self.config.model = self.config.model or "intfloat/e5-mistral-7b-instruct"
+        self.config.embedding_dims = self.config.embedding_dims or 4096
```

Comment:
Let's remove the default from here. I think it would be better if the developer passes which model they want to use.